Microsoft.Toolkit.Parsers/Markdown/MarkdownDocument.cs

```diff
@@ -13,14 +13,15 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
-using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Helpers;
+using Microsoft.Toolkit.Parsers.Markdown.Helpers;
+using Microsoft.Toolkit.Parsers.Markdown.Parse;
 
-namespace Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Parse
+namespace Microsoft.Toolkit.Parsers.Markdown
```

Comment:
same here: re breaking changes

Reply:
This isn't a breaking change as all of it's containing members were internal.